src/main/java/car_racing/domain/model/RaceResults.java

```diff
@@ -0,0 +1,46 @@
+package car_racing.domain.model;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class RaceResults {
+    private static final String RACE_RESULT_DELIMITER = " : ";
+
+    private final List<Car> cars;
+
+    public RaceResults(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public List<String> getRaceResultOfNthRound(int n) {
+        List<String> result = new ArrayList<>();
+        for (Car car : cars) {
+            result.add(getRaceResultOfNthRoundOfEachCar(n, car));
+        }
+        return result;
+    }
+
+    private String getRaceResultOfNthRoundOfEachCar(int n, Car car) {
+        RaceResult raceResult = car.getRaceResult();
+
+        StringBuilder sb = new StringBuilder();
+        sb.append(car.getName())
+                .append(RACE_RESULT_DELIMITER)
+                .append(raceResult.getPositionOfNthRound(n));
+
+        return sb.toString();
+    }
+
+    public List<Car> getWinners() {
+        if (cars.isEmpty()) return Collections.emptyList();
+
+        int maxDistance = cars.stream().mapToInt(Car::getDistance).max().orElse(0);
```

Comment:
```suggestion
        int maxDistance = cars.stream()
        .mapToInt(Car::getDistance)
        .max()
        .orElse(0);
```

이렇게 쓰면 가독성 측면에서 위에서 아래느낌으로 읽히기가 쉽습니다.